interfaces/builtin/spi.go

```diff
@@ -84,7 +84,7 @@ func (iface *spiInterface) AppArmorConnectedPlug(spec *apparmor.Specification, p
 		return nil
 	}
 	spec.AddSnippet(fmt.Sprintf("%s rw,", path))
-	spec.AddSnippet(fmt.Sprintf("/sys/devices/platform/**/**.spi/**/%s/** rw,", strings.TrimPrefix(path, "/dev/")))
+	spec.AddParametricSnippet("/sys/devices/platform/**/**.spi/**/###PARAM###/** rw,", strings.TrimPrefix(path, "/dev/"))
```

Comment:
Like above, can we use (enforced by spiDevPattern):
```
deviceName := strings.TrimPrefix(path, "/dev/")
spec.AddParametricSnippet("/sys/devices/platform/**/**.spi/**/spidev###PARAM###/** rw,", deviceName[6:]) # len("spidev") is 6
```

Reply:
Done, in a slightly different way but the same effect.